pkg/apis/core/validation/validation_test.go

```diff
@@ -16659,9 +16671,11 @@ func TestValidateReplicationControllerUpdate(t *testing.T) {
 	for _, tc := range successCases {
 		tc.old.ObjectMeta.ResourceVersion = "1"
 		tc.update.ObjectMeta.ResourceVersion = "1"
-		if errs := ValidateReplicationControllerUpdate(&tc.update, &tc.old, PodValidationOptions{}); len(errs) != 0 {
+		errs := ValidateReplicationControllerUpdate(&tc.update, &tc.old, PodValidationOptions{})
```

Comment:
curious why this change?

Reply:
When I picked up commits, this is what we I got from the dev branch: https://github.com/jpbetz/kubernetes/blob/21600930a778dd13decc388f5e4ed956e8c31439/pkg/apis/core/validation/validation_test.go#L16640

I personally prefer the `if` clause, but I biased toward consistency with the dev branch.

Reply:
I t looks like it came from a series of changes which were made and removed (the gate checking that used to be here).  I will send a fix to dev, but we can pick it up later when we do stragglers.

Reply:
OK, After consolidating the group specific testing to strategy and the generic testing to pkg/api/testing I don't need any changes to validation_test.go so I'm dropping it from this PR.